man/man5/mash_client.conf.5

```diff
@@ -0,0 +1,130 @@
+.\" Process this file with
+.\" groff -man -Tascii mash_client.conf.5
+.\"
+.TH mash_client.conf.5
+.SH NAME
+profile.yaml \- Configuration file for the mash client
```

Comment:
It may be more clear to wrap this in some way like {profile}.yaml to denote that profile is a variable based on `--profile` which has a default value of default (default.yaml).